modules/ROOT/pages/ai-azure.adoc

```diff
@@ -0,0 +1,211 @@
+= Azure AI integration guide
+:pluginname: AI Assistant
+:plugincode: ai
+:plugincategory: premium
+:navtitle: Azure AI integration guide
+:description_short: {pluginname} with Azure OpenAI Service
+:description: A guide for integrating {pluginname} plugin using Azure OpenAI Service.
+:keywords: example, demo, custom, plugin, ai, assistant, guide, azure, openai, service
+
+include::partial$misc/admon-ai-pricing.adoc[]
+
+== Introduction
+
+This guide provides instructions for integrating the {pluginname} plugin using the https://learn.microsoft.com/en-us/azure/ai-services/openai/[Azure OpenAI Service] in {productname}.
+
+Azure OpenAI Service provides REST API access to OpenAI's language models with Azure specific features such as private networking, regional availability, and responsible AI content filtering.
+
+To learn more about the difference between string and streaming responses, see xref:ai.adoc#the-respondwith-object[The `respondWith` object] on the plugin page.
+
+== Prerequisites
+
+Before beginning, make sure an OpenAI model has been set up in Azure. The model's endpoint URL and API key are required for integration. For more information, see the https://learn.microsoft.com/en-us/azure/ai-services/openai/chatgpt-quickstart?tabs=command-line%2Cpython-new&pivots=rest-api[Quick Start Guide] provided by Azure.
+
+include::partial$misc/admon-ai-proxy.adoc[]
+
+== String response
+
+This example demonstrates how to integrate the {pluginname} plugin with the https://learn.microsoft.com/en-us/azure/ai-services/openai/reference#chat-completions[Azure OpenAI Service Chat completions API] to generate a string response.
+
+[source,js]
+----
+// This example stores the API key in the client side integration. This is not recommended for any purpose.
+// Instead, an alternate method for retrieving the API key should be used.
+const AZURE_OPENAI_API_KEY = '<INSERT_API_KEY_HERE>';
+const AZURE_OPENAI_ENDPOINT = '<INSERT_ENDPOINT_URL_HERE>'; // e.g. https://{your-resource-name}.openai.azure.com/openai/deployments/{deployment-id}/chat/completions?api-version={api-version}
+
+const ai_request = (request, respondWith) => {
+  const azureOpenAiOptions = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'api-key': AZURE_OPENAI_API_KEY
```

Comment:
```suggestion
      'api-key': `Bearer ${AZURE_OPENAI_API_KEY}`
```
Just want confirmation that this is not the correct format.

Reply:
The Azure quick start guide indicates using an `api-key` header so have just followed that: https://learn.microsoft.com/en-us/azure/ai-services/openai/quickstart?tabs=command-line%2Cpython-new&pivots=rest-api#rest-api